Misc review fixes (#4801)
qgis/QGIS-Documentation: source/docs/user_manual/working_with_vector/editing_geometry_attributes.rst

Changing the direction of a line geometry can be useful for
cartographical purposes or when preparing for network analysis.

To change a line direction:

#. Activate the reverse line tool by clicking |reverseLine|
   :sup:`Reverse line`.

> #. Click the line with the cursor. The direction of the line

Reviewer: ```suggestion
#. Click on the line. The direction of the line
```
